docs/topics/db/queries.txt

```diff
@@ -1790,6 +1790,14 @@ methods::
 
     b.entry_set(manager="entries").is_published()
 
+.. admonition:: Interaction with prefetching
+
+   :meth:`~django.db.models.query.QuerySet.prefetch_related` on a reverse
+   relation always uses the default manager. Therefore, using a custom manager
+   in conjunction with  
+   :meth:`~django.db.models.query.QuerySet.prefetch_related`
+   will not produce expected results.
```

Comment:
We should document an alternative instead of documenting this as a bug, e.g.
```
Use :class:`Prefetch() <django.db.models.Prefetch>`, if you want to prefetch
related objects using a custom reverse manager. For example::

    from django.db.models import Prefetch

    prefetch_manager = Prefetch("entry_set", queryset=Entry.entries.all())
    Blog.objects.prefetch_related(prefetch_manager)
```

Reply:
... so I'd remove _"Therefore, using a custom manager ... will not produce expected results."_ and add something similar to the :point_up: